src/StoreApi/Routes/V1/AbstractCartRoute.php

```diff
@@ -232,7 +234,7 @@ protected function cart_updated( \WP_REST_Request $request ) {
 		$draft_order = $this->get_draft_order();
 
 		if ( $draft_order ) {
-			$this->order_controller->update_order_from_cart( $draft_order );
+			$this->order_controller->update_order_from_cart( $draft_order, false );
```

Comment:
I think it'd be nice to add a comment here for why we're not recalculating totals, it might be confusing in the future when we're trying to work out why